qa-physics/buildChargeTree.groovy

```diff
@@ -60,6 +94,8 @@ dataFile.eachLine { line ->
         'livetime':livetime
       ]}
     )
+    helState.eachWithIndex { it, i -> T.addLeaf(chargeTree,[runnum,binnum,'helicity',it],{helicity[i]})}
```

Comment:
For clarity, the key should indicate this is a charge, something like
```suggestion
    helState.eachWithIndex { it, i -> T.addLeaf(chargeTree,[runnum,binnum,'fcChargeHelicity',it],{helicity[i]})}
```
(or a better name)